modules/javafx.graphics/src/main/java/javafx/css/StyleConverter.java

```diff
@@ -86,6 +86,12 @@
  */
 public class StyleConverter<F, T> {
 
+    /**
+     * Creates a default {@code StyleConverter}.
+     */
+    public StyleConverter() {
+    }
+
     /**
```

Comment:
Looks like a constructor is fine here if the predefined factories are not suitable, but I'm not familiar with this.

Reply:
This one needs to be public since it is subclassed in many other classes.